features/FEATURE_UVISOR/includes/uvisor/api/inc/uvisor-lib.h

```diff
@@ -17,6 +17,8 @@
 #ifndef __UVISOR_API_UVISOR_LIB_H__
 #define __UVISOR_API_UVISOR_LIB_H__
 
+#include "includes/uvisor-lib/uvisor-lib.h"
```

Comment:
Yeah, removing will cause compilation errors. I was curious which files were having errors.

Thanks for the fix. :)